smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -91,6 +89,7 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
     } finally {
       // Check all mock expectations are satisfied.
       mockControl.$verifyAll();
+      assertTrue(chromeApiProvider.isDisposed());
```

Comment:
Let's do it inside the `try {` block. `$verifyAll()` belongs there as well I think.

My reasoning is: the `try-finally` block is intended to make sure no globals leak between test cases. Hence let's try to make `finally` as bulletproof as possible, with everything else sitting in the `try` block.

Reply:
thanks, fixed